ui/src/data_explorer/components/DatabaseList.js

```diff
@@ -59,10 +61,10 @@ const DatabaseList = React.createClass({
   },
 
   render() {
-    const {onChooseNamespace, query} = this.props
+    const {query, onChooseNamespace} = this.props
 
     return (
-      <div className="query-builder--column">
+      <div className="query-builder--column" style={{flex: '1 0 0'}}>
```

Comment:
it seems like a common set of classes to have anyway... but we may want to discuss / spike how we really want to do styles. I'm not a fan of sass.